script/script.go

```diff
@@ -232,33 +242,36 @@ func serveRequest(f filters.FilterContext) func(*lua.LState) int {
 		t := s.Get(-1)
 		r, ok := t.(*lua.LTable)
 		if !ok {
-			s.Push(lua.LString("invalid type, need a table"))
-			return 1
+			s.RaiseError("unsupported type %v, need a table", t.Type())
```

Comment:
You could also use `%T` with `t`, but it's also fine like this if t is not nil (?)

Reply:
It produces e.g. `unsupported type string, need a table`, `unsupported type nil, need a table`

And
```go
s.RaiseError("unsupported type %T, need a table", t)
```
produces `unsupported type lua.LString, need a table`, `unsupported type *lua.LNilType, need a table`.

I like as it is but will add a test anyway.

Reply:
ok then let's merge it like it is now

Reply:
I resolved the rest of the comments refering to here.
I think you have your point in showing me the log message and I agree the current one with using .Type() is nicer.